Resources/Prototypes/_NF/Entities/Structures/Machines/cryo_sleep_pod.yml

```diff
@@ -15,11 +15,11 @@
     bodyType: Static
   - type: MaterialStorage
   - type: Appearance
-#  - type: Climbable
+  - type: Climbable
```

Comment:
Unless you can confirm the bug was fixed this is really not needed.

Reply:
I need the exact info on how that bug occurs. Is it the bug that pushes you into a wall when you climb a thing? That has happened to me with tables and different climbable objects.

This is necessary to let people get out of a cryopod (even if they simply decline the cryo dialog), the only alternative I see is making them non-collidable, like chairs and other stuff you put yourself on top of.

Reply:
I don't have a video but it's making you permantly able to pass into walls at anytime if you walk into walls after you go on top of it,

You can update the machine layer to make them walk over to fix it and remove the climbable

Reply:
I'm not sure, but it might have been happening because the entering player would never be removed from the body container of a cryo chamber